src/components/mobile/molecules/GameTagModal/GameTagModal.tsx

```diff
@@ -32,6 +33,12 @@ const GameTagModal = ({
 
   const handleTagSubmit = () => {
     if (currentMainTag) {
+      const gameValidation = validateGameTag(form);
+
+      if (!gameValidation.isValid) {
+        return;
+      }

```

Comment:
```suggestion
const handleTagSubmit = () => {
  if (!currentMainTag) return;

  const { isValid } = validateGameTag(form);
  if (!isValid) return;

  submitGame();
  onClose?.();
};
```

early return 사용을 해도 좋아보여요

Reply:
[e7c0e2f](https://github.com/JECT-Study/PICK-O-Client/pull/306/commits/e7c0e2fa2b298e4e62d18f8b4f34190b233840a1) 에서 수정했습니다!!